ClassHomework/src/Harry_Potter_Reader/WordCounter.java

```diff
@@ -0,0 +1,54 @@
+package Harry_Potter_Reader;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Scanner;
+
+public class WordCounter {
+	static HashMap<String, Integer> wordCount = new HashMap<String, Integer>();
```

Comment:
good. You went far and beyond here. We haven't covered Maps yet but this would be the best way to implement this type of question.